src/Resolvers/Collections.ts

```diff
@@ -13,7 +13,7 @@ export class CollectionsResolver {
     @Arg("artistID", { nullable: true }) artistID: string,
     @Arg("showOnEditorial", { nullable: true }) showOnEditorial: boolean,
     @Arg("size", () => Int, { nullable: true }) size: number,
-    @Arg("randomize", { nullable: true }) randomize: boolean
+    @Arg("randomizationSeed", { nullable: true }) randomizationSeed: string
```

Comment:
In terms of the API, could we actually go with the following 
```graphql
collections(randomize: true, seed: "arbitrary_string") {
  id
}
```
as I think it makes the intent clearer to the consumer. We can also add a description to those arguments

e.g.
```
@Arg("randomize", () => Boolean, { nullable: true, description: "Return a random sample of collection. Must specify size and seed" }) randomize: boolean
```